docs/source/custom/installation.rst

```diff
@@ -157,15 +157,30 @@ it is likely you will need to specify a legacy compiling configuration by changi
 
 2. Configure your environment by pointing the SIXS_DIR variable to point to your installation directory.
 
-3. Download the `pre-trained sRTMnet neural network <https://zenodo.org/record/4096627>`_, and (for the example below)
-point the environment variable EMULATOR_PATH to the base unzipped path.
+3. Download the `pre-trained sRTMnet neural network hdf5 file <https://avng.jpl.nasa.gov/pub/PBrodrick/isofit/sRTMnet_v100.h5>`_,
+the sRTMnet auxiliary file that is included in a `Zenodo-hosted zip file <https://zenodo.org/record/4096627/files/sRTMnet_v100.zip>`_, and (for the example below)
```

Comment:
For the time being, would it make sense to point users directly to downloading `https://avng.jpl.nasa.gov/pub/PBrodrick/isofit/sRTMnet_v100_aux.npz` instead of the large zip? Makes life a little easier etc...

Reply:
Absolutely! I somehow lost it in my mind that we uploaded the file there as well. Will update the docs!